src/main/java/com/github/manolo8/darkbot/core/entities/Npc.java

```diff
@@ -76,16 +76,31 @@ public NpcInfoImpl(HeroItemsAPI items, SettingsProxy settingsProxy) {
             this.items = items;
             this.settingsProxy = settingsProxy;
         }
+        
+        @Override
+        public boolean setShouldKill(boolean shouldKill) {
```

Comment:
setters should be void, not return the value being set
```suggestion
        public void setShouldKill(boolean shouldKill) {
```
if in the api it was defined like this, it was wrong and should be changed